NPE in HeapMonitorPane

```
Unable to wait until the task is completed
java.util.concurrent.ExecutionException: org.eclipse.e4.core.di.InjectionException: java.lang.NullPointerException: Cannot invoke "com.osgifx.console.supervisor.Supervisor.getAgent()" because "this.supervisor" is null
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at org.eclipse.fx.ui.controls.internal.FXThreadSynchronizeImpl.syncExec(FXThreadSynchronizeImpl.java:114)
	at org.eclipse.fx.ui.workbench.fx.internal.UISynchronizeImpl.syncExec(UISynchronizeImpl.java:51)
	at org.eclipse.e4.ui.internal.di.UIEventObjectSupplier$UIEventHandler.handleEvent(UIEventObjectSupplier.java:64)
	at org.eclipse.equinox.internal.event.EventHandlerWrapper.handleEvent(EventHandlerWrapper.java:205)
	at org.eclipse.equinox.internal.event.EventHandlerTracker.dispatchEvent(EventHandlerTracker.java:203)
	at org.eclipse.equinox.internal.event.EventHandlerTracker.dispatchEvent(EventHandlerTracker.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:345)
Caused by: org.eclipse.e4.core.di.InjectionException: java.lang.NullPointerException: Cannot invoke "com.osgifx.console.supervisor.Supervisor.getAgent()" because "this.supervisor" is null
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:68)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:457)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:456)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
Caused by: java.lang.NullPointerException: Cannot invoke "com.osgifx.console.supervisor.Supervisor.getAgent()" because "this.supervisor" is null
	at com.osgifx.console.ui.heap.HeapMonitorPane.createMainContent(HeapMonitorPane.java:125)
	at com.osgifx.console.ui.heap.HeapMonitorPane.init(HeapMonitorPane.java:92)
	at com.osgifx.console.ui.heap.HeapMonitorFxUI.updateOnAgentDisconnectedEvent(HeapMonitorFxUI.java:73)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:58)
	... 6 common frames omitted
```

--- a/com.osgifx.console.ui.heap/src/main/java/com/osgifx/console/ui/heap/HeapMonitorPane.java
+++ b/com.osgifx.console.ui.heap/src/main/java/com/osgifx/console/ui/heap/HeapMonitorPane.java
@@ -122,9 +122,13 @@ public final class HeapMonitorPane extends BorderPane {
         separator.setPrefHeight(2);
         vBoxChildren.add(separator);
 
-        final var agent = supervisor.getAgent();
-        if (agent != null && agent.getHeapUsage() != null) {
-            for (final XMemoryPoolMXBean mpBean : agent.getHeapUsage().memoryPoolBeans) {
+        Agent agent = null;
+        if (supervisor == null || (agent = supervisor.getAgent()) == null) {
+            return box;
+        }
+        final var heapUsage = agent.getHeapUsage();
+        if (heapUsage != null) {
+            for (final XMemoryPoolMXBean mpBean : heapUsage.memoryPoolBeans) {
                 if ("HEAP".equals(mpBean.type)) {
                     final var memoryUsageChart = new HeapMonitorChart(mpBean.name,
                             getMemoryUsagedByMemoryPoolBean(mpBean), now);

